src/fileappender.cxx

```diff
@@ -1431,10 +1431,10 @@ TimeBasedRollingFileAppender::clean(Time time)
     }
 
     Time::duration period = getRolloverPeriodDuration();
-    long periods = long(interval.count () / period.count ());
+    auto periods = long(interval.count () / period.count ());
 
     helpers::LogLog & loglog = helpers::getLogLog();
-    for (long i = 0; i < periods; i++)
+    for (auto i = 0; i < periods; i++)
```

Comment:
This changes the `i` type to an `int`, AFAIK and it is not an improvement. `long` is trivially easy to understand.